### ISSUE TITLE
Server Script Compile Error (See Console)

### ISSUE THREAD

**Issue by FatheredPuma81** [xoxor4d/t4-rtx#4]
Followed manual update instructions and it results in this.
After a but if testing the last working build is Readme.

**Comment by xoxor4d** [comment]
You are most likely running the game with developer 1 or developer_script 1 right? (From the commandline).
If I understood you correctly, you downloaded the latest nightly? Do you have the `_debug.gsc` in `t4rtx/_assets/maps`?

**Comment by FatheredPuma81** [comment]
Downloaded the latest release (https://github.com/xoxor4d/t4-rtx/releases)
Downloaded the latest actions (https://github.com/xoxor4d/t4-rtx/actions/runs/10568265932)
Copied contents of release into game folder
Launched game, opened NDU, closed game (Otherwise it crashes on startup on the final step)
Deleted nvapi
Copied contents of actions and actions/assets into game folder
Launch game, choose any map, "Server Script Compile Error (See Console)" after a bit of loading

Yea debug.gsc is in "\Call of Duty World at War\t4rtx\_assets\maps\"

I also tried every action as far back as (https://github.com/xoxor4d/t4-rtx/actions/runs/10022349333) and the only two that don't work is the latest and one before that (https://github.com/xoxor4d/t4-rtx/actions/runs/10521583546)

**Comment by xoxor4d** [comment]
Can you open the full console with shift + console key to see the full error message? If its also a generic one, could you launch the game with 
**+set developer 1 +set developer_script 1**

**Comment by FatheredPuma81** [comment]
![Screenshot_1](https://github.com/user-attachments/assets/2a70b7ea-63dd-454a-a5a0-22a8c8c772e2)
Here's an SS of the console log.

**Comment by FatheredPuma81** [comment]
Here's a screenshot with "+set developer 1 +set developer_script 1" on 
![Screenshot_1](https://github.com/user-attachments/assets/30727dd6-5bd8-47e3-821b-225e92abb7e1)

**Comment by xoxor4d** [comment]
Aaaah yes thank you! Will fix that tomorrow :)

**Comment by FatheredPuma81** [comment]
> Aaaah yes thank you! Will fix that tomorrow :)

Already did on my end by just editing the text and now it outright crashes with 
```
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1142)
 vehicles.y = 320;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1143)
 vehicles.label = &"DEBUG_VEHICLES";
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1144)
 vehicles.alpha = 0;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1145)
 vehicles.fontscale = 1.2;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
Debug hudelems are not built into this exe: 
Debug hudelems are not built into this exe: (file 'maps/_debug.gsc', line 1148)
 total = NewDebugHudElem();
 
  
   
    
     
      
       
        
         
         *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1149)
 total.alignX = "left";
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1150)
 total.alignY = "middle";
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1151)
 total.x = 620;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1152)
 total.y = 335;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1153)
 total.label = &"DEBUG_TOTAL";
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1154)
 total.alpha = 0;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************

******* script runtime error *******
undefined is not a field object: 
undefined is not a field object: (file 'maps/_debug.gsc', line 1155)
 total.fontscale = 1.2;
 
 *
Error: called from:
(file 'maps/_debug.gsc', line 725)
 thread debug_character_count();
 
  
   
    
     
      
       
        
        *
Error: started from:
(file 'maps/_debug.gsc', line 700)
 waittillframeend; // for vars to get init'd elsewhere
 
 *
Error: ************************************
-----------------------------------
```

**Comment by FatheredPuma81** [comment]
Yea I'd suggest just adding something to the "B. Manual way but always up-to-date:" to clear up any confusion. I was under the assumption that everything there would have been rudimentarily tested to at least get into the map. This is far out of my league.

**Comment by xoxor4d** [comment]
I was running into similar issues when running in developer mode. That's why I've shipped a custom debug.gsc that disabled functionality similar to the error you are getting. I think that's a base game bug.

Anyhow, running the game with developer 0 and developer_script 0 should not result in the error being displayed?

**Comment by FatheredPuma81** [comment]
Only enabled developer mode to provide the screenshot. After deleting the printConsole command that was preventing me from even loading a map the game now crashes after loading any map and spits out an Unhandled Exception error in the crash manager. Doesn't matter if I have developer mode enabled or disabled.

**Comment by xoxor4d** [comment]
The errors related to the gsc files should be fixed with the latest commit.
Unsure about the exception you are getting. Steam version of the game or retail? 
You could try to grab my latest [bridge-remix nightly](https://github.com/xoxor4d/bridge-remix/actions) (with the required api features) and the latest official [dxvk-remix nightly](https://github.com/NVIDIAGameWorks/dxvk-remix/actions)

**Comment by FatheredPuma81** [comment]
Latest bridge is from 3 weeks ago btw not sure if that's right. Anyways I grabbed the latest everything and tried various different configurations and this is what I ended up with:
Bridge + Binaries + DXVK +/- Full release (February)= RTX Remix Bridge Error! on map load
Full release (February) + Binaries +/- DXVK Remix = Unhandled Exception (game crash) on map load
Full release (February) + DXVK Remix = Runs fine with more features
Bridge +/- Full release (February) = RTX Remix Bridge Error! on launch
Bridge + DXVK = Runs with old visuals and glitched out lighting
